validators/ununifid-installation-and-setup.md

```diff
@@ -1,3 +1,57 @@
-# ununifid Installation and Setup 
-
-Here for you to write the content. (you can delete this, of course.)
+UnUniFi Installation and Setup 
+=
+Instruction to install the ununifid binary
+### Choose an Operating System
+The operating system you use for your node is entirely your personal preference. You will be able to compile the ununifid daemon on most modern linux distributions and recent versions of macOS
+> For the tutorial, it is assumed that you are using an Ubuntu LTS release.
+> If you have chosen a different operating system, you will need to modify your commands to suit your operating system.
+### Requirements
+Validator Node Server
+- OS: Ubuntu 20.04
+- Memory: 8 GB or more
+- Storage: SSD 160 GB or more
+- The following ports: `26656` must be open for peer to peer communication between nodes.
+### Install pre-requisites
+```Bash
+# update the local package list and install any available upgrades
+sudo apt-get update && sudo apt upgrade -y
+# install toolchain and ensure accurate time synchronization
+sudo apt-get install make build-essential gcc git jq chrony -y
+```
+### Install Go
+Follow the instructions [here](https://go.dev/doc/install) to install Go.
+For an Ubuntu LTS, you can probably use:
+```Bash
+# Please install Go v1.18 or later
+# from $HOME dir
+$ wget https://golang.org/dl/go1.18.2.linux-amd64.tar.gz
+$ sudo rm -rf /usr/local/go
+$ sudo tar -C /usr/local -xzf go1.18.2.linux-amd64.tar.gz
+$ go version
+go version go1.18.2 linux/amd64
```

Comment:
```suggestion
# Please install Go v1.17
# from $HOME dir
$ wget https://go.dev/dl/go1.17.linux-amd64.tar.gz
$ sudo rm -rf /usr/local/go
$ sudo tar -C /usr/local -xzf go1.17.linux-amd64.tar.gz
$ go version
go version go1.17 linux/amd64
```
I would like the go version to match the 1.17 indicated in the README of the [chain repository](https://github.com/UnUniFi/chain#install).
So please make this correction.